recipe/LICENSE

```diff
@@ -0,0 +1,13 @@
+Copyright 2017 Lee McCuller
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
```

Comment:
This file is bundled with the git archive distribution, do we really need a separate copy here?

Reply:
I think you are right. It wasn't clear from the documentation and checklist, but I suppose it captures it from the sources.